### ISSUE TITLE
Add fig-alt support to rgl graphs

### ISSUE THREAD

**Issue by dmurdoch** [dmurdoch/rgl#341]
#336 correctly noted that the vignette had base graphics without alt text.  However, all rgl plots also lack alt text, because the widget doesn't support it.  That should be added.  

It won't be trivial, because rgl figures are normally `canvas` elements, not `img` elements, and `canvas` doesn't support alt text.  But there are workarounds, e.g. https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Basic_usage .

**Comment by dmurdoch** [comment]
The first attempt wasn't visible, so there's a new attempt in #343 .

**Comment by dmurdoch** [comment]
I believe this is all done now, subject to `knitr` accepting my pull request:  https://github.com/yihui/knitr/pull/2243 .  I'll leave the issue open until that is resolved.
